packages/contracts-periphery/contracts/L2/L2ERC721Bridge.sol

```diff
@@ -241,6 +262,39 @@ contract L2ERC721Bridge is Semver, CrossDomainEnabled, Initializable {
         }
     }
 
+    /**
+     * @notice Completes an outbound token transfer. Public function, but can only be called by
+     *         this contract. It's security critical that there be absolutely no way for anyone to
+     *         trigger this function, except by explicit trigger within this contract. Used as a
+     *         simple way to be able to try/catch any type of revert that could occur during an
+     *         ERC721 mint/transfer.
+     *
+     * @param _localToken  Address of the ERC721 on this chain.
+     * @param _remoteToken Address of the corresponding token on the remote chain.
+     * @param _to          Address of the receiver.
+     * @param _tokenId     ID of the token being deposited.
+     */
+    function completeOutboundTransfer(
+        address _localToken,
+        address _remoteToken,
+        address _to,
+        uint256 _tokenId
+    ) external onlySelf {
```

Comment:
I noticed that the StandardBridge has this extra check: 
https://github.com/ethereum-optimism/optimism/blob/111bdcdaf4c7ac7659936c1c78aaad3c6b7b9c91/packages/contracts-bedrock/contracts/universal/StandardBridge.sol#L375

It seems a little paranoid, but maybe I'm missing something behind the motivation for it. @smartcontracts do you think it's needed here?